src/main/java/com/minecolonies/rankup/modules/timing/command/addCommand.java

```diff
@@ -36,7 +36,7 @@
     @Override
     public Optional<Text> getDescription()
     {
-        return Optional.of(Text.of("Allows players with the rankup.timing.base to add time to another players playtime. \n Usage: /ru add [time] {player}"));
+        return Optional.of(Text.of("Allows players with the rankup.timing.base to add time to another players playtime. \n Usage: /ru add [time] " + Constants.PlayerInfo.PLAYER_NAME + ""));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 189 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS00103)